Loop/Managers/DeviceAlert/DeviceAlertManager.swift

```diff
@@ -76,28 +76,28 @@ extension DeviceAlertManager: DeviceAlertPresenter {
 extension DeviceAlertManager {
     
     public static func soundURL(for alert: DeviceAlert) -> URL? {
-        guard let soundName = alert.soundName else { return nil }
-        return soundURL(managerIdentifier: alert.identifier.managerIdentifier, soundName: soundName)
+        return soundURL(managerIdentifier: alert.identifier.managerIdentifier, sound: alert.sound)
     }
     
-    private static func soundURL(managerIdentifier: String, soundName: DeviceAlert.SoundName) -> URL? {
-        guard soundName != .vibrate && soundName != .silence else { return nil }
+    private static func soundURL(managerIdentifier: String, sound: DeviceAlert.Sound?) -> URL? {
```

Comment:
Why did the `sound` argument become an Optional?

Reply:
It's minor, but it is because the call above passes in `alert.sound` which is optional.  It may be a tad cleaner to deal with it there...good point.